Formula/llvm@11.rb

```diff
@@ -132,6 +133,9 @@ def install
       -DLLDB_PYTHON_RELATIVE_PATH=libexec/#{site_packages}
       -DLIBOMP_INSTALL_ALIASES=OFF
       -DCLANG_PYTHON_BINDINGS_VERSIONS=#{py_ver}
+      -DPACKAGE_VENDOR=#{tap.user}
+      -DPACKAGE_BUGREPORT=#{tap.issues_url}
+      -DCLANG_VENDOR_UTI=org.#
```

Comment:
```suggestion
      -DCLANG_VENDOR_UTI=org.#{tap.user.downcase}.clang
```

Reply:
Whoops, had a copy/paste issue with my editor for that one.